### PR TITLE
moving content for volume mounts from cf docs into spec

### PR DESCRIPTION
**Pull request by shalako**
The spec for the `volume_mounts` field in the bind response was documented as a separate file, then removed from the CF docs by accident so it was missed when we migrated the spec. 

We have since added the doc back to the CF docs: http://docs.cloudfoundry.org/services/volume-services.html. Once this is merged, we'll remove the doc from the CF docs. I will submit a PR to the 2.12 branch as well.

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -640,9 +640,9 @@ did not include a `"requires":["route_forwarding"]` property.
 
 There are a class of services that provide network storage to applications
 via volume mounts in the application container. A create binding response
-from one of these services MUST include a `volume_mount`.
+from one of these services MUST include `volume_mounts`.
```
Comment (pmorie) [inline spec.md]: I'm cool with moving this information in, but we might want to add a note saying that for route and volume services, users should check to see that their platforms support those services.

Comment (shalako) [inline spec.md]: I'd like such a note to be accompanied by a link to a section in profile.md that specifies what features are (or are not) supported by example platforms. Can we do that in a separate PR?

**Comment by duglin** [comment]
LGTM

**Comment by angarg12** [comment]
LGTM

**Comment by avade** [comment]
lgtm

**Comment by duglin** [comment]
one more l.g.t.m needed

**Comment by zrob** [comment]
LGTM
